Library/Formula/llvm.rb

```diff
@@ -211,6 +211,11 @@ def install
 
     args << "-DLINK_POLLY_INTO_TOOLS:Bool=ON" if build.with? "polly"
 
+    args << "-DLIBCLANG_BUILD_STATIC:Bool=ON" if build.with? "clang"
+
+    args << "-DLIBCXX_ENABLE_SHARED:Bool=OFF"    if build.with? "libcxx"
+    args << "-DLIBCXXABI_ENABLE_SHARED:Bool=OFF" if build.with? "libcxx"
```

Comment:
Please don't use this style if both arguments are triggered by the same formula option. Something like the following is preferable in these cases (possibly without the line break, if the result comfortably fits on a single line):

```ruby
if build.with? "libcxx"
  args << "-DLIBCXX_ENABLE_SHARED:Bool=OFF"
       << "-DLIBCXXABI_ENABLE_SHARED:Bool=OFF"
end
```